### DISCUSSION TITLE
base sales from base sales model

### DISCUSSION THREAD

**Discussion by robbenyam** [sibylhe/mmm_stan#66] [Q&A]
The "Readme" said: predict base sales (X_ctrl) as an input variable to MMM, this represents the baseline sales trend without any marketing activities.

the predict base sales derived from base sales model , and this model is based on the relationship between "sales" and "control varibles", however, data of "sales" are real data which influenced by both of "control varibles" and "media varibles". 

Is it reasonable to say that predict base sales represents the baseline without any marketing activities?

Thanks

**Comment by ChengchengCai006** [comment]
same question, hope someone to reply it!

**Reply by Vineet3012** [reply]
please can you clarify your question? baseline sales after mean centering? or base sales (the one only due to base vars)?
